assets/js/types/type-defs/contexts.ts

```diff
@@ -13,6 +13,12 @@ export type ValidationData = {
 	) => void;
 };
 
+export type ProductQueryContext = {
```

Comment:
Is there a reason why this is a `type` as opposed to `interface` like the others you have created?

Reply:
Addressed in ad2b4decd.